source/extra/examples/LifecycleManagementUser.json

```diff
@@ -15,7 +15,7 @@
             "Action": [
                "s3:GetObject",
                "s3:PutObject",
-               "s3:DeleteObject"
+               "s3:DeleteObject",
```

Comment:
I don't think this is a valid JSON with this extra comma

Reply:
Oops thank you!